api/src/main/java/org/openmrs/module/m2sysbiometrics/client/M2SysV105Client.java

```diff
@@ -107,8 +108,16 @@ public BiometricSubject updateSubjectId(String oldId, String newId) {
 
     @Override
     public List<BiometricMatch> search() {
-        M2SysCaptureResponse capture = scanDoubleFingers();
-        return searchService.search(capture, localBioServerClient);
+        M2SysCaptureResponse fingerScan = scanDoubleFingers();
+        List<BiometricMatch> results = searchService.search(fingerScan, localBioServerClient);
+        if (CollectionUtils.isEmpty(results)) {
+            BiometricMatch nationalResult = searchService.findMostAdequate(fingerScan, nationalBioServerClient);
```

Comment:
why can't we use search here?

Moreover - is the patient imported from the Mpi if this is a national hit? Or will that be a separate commit?